projects/api-client/src/model/mainBranch.ts

```diff
@@ -2,7 +2,7 @@
  * Wharf main API
  * Wharf backend API that manages data storage for projects, builds, providers, etc.
  *
- * OpenAPI spec version: v4.1.0
+ * OpenAPI spec version: v4.2.0
```

Comment:
This doesn't seem right. Maybe this file is no longer needed? I think all of the code inside the src dirs are generated, so it should be fine to run the following:

```sh
rm -rfv projects/*/src/*
```

Reply:
Yeah I was wondering about those, thought it might've had something to do with deprecated things.
Regenerating after running `rm -rfv projects/*/src/*`.